stage/LibUsbDotNet/WinUsb/Internal/WinUsbAPI.cs

```diff
@@ -186,8 +186,18 @@ internal class WinUsbAPI : UsbApiBase
                                            int BufferLength,
                                            out int LengthTransferred) { return WinUsb_GetDescriptor(InterfaceHandle, DescriptorType, Index, LanguageID, Buffer, BufferLength, out LengthTransferred); }
 
-        public override bool GetOverlappedResult(SafeHandle InterfaceHandle, IntPtr pOVERLAPPED, out int numberOfBytesTransferred, bool Wait) { return WinUsb_GetOverlappedResult(InterfaceHandle, pOVERLAPPED, out numberOfBytesTransferred, Wait); }
-
+        public override bool GetOverlappedResult(SafeHandle InterfaceHandle, IntPtr pOVERLAPPED, out int numberOfBytesTransferred, bool Wait) 

```

Comment:
The whitespace is off here, can you use spaces instead of tabs (like in the rest of the project)? Thanks!

Reply:
Yep, sorry I forgot to check the whitespace before the commit.